CHANGELOG.md

```diff
@@ -2,13 +2,13 @@
 
 ## [Unreleased]
 
-## [3.0.0] 2023-02-28
+## [3.0.0] 2023-03-07
```

Comment:
I presume the Git tag (from the failed release) will be updated/moved once this is released? :-)
https://github.com/heroku/heroku-gradle-plugin/releases/tag/v3.0.0

Reply:
Thanks for the calling this out explicitly! Yeah, I'll take care of this.